x/liquidity/client/cli/tx.go

```diff
@@ -90,7 +91,67 @@ New liquidity pools can be created only for coin combinations that do not alread
 				return fmt.Errorf("the number of deposit coins must be two in pool-type 1")
 			}
 
-			msg := types.NewMsgCreatePool(poolCreator, uint32(poolTypeId), depositCoins)
+			var swapFeeRate *sdk.Dec
+			if len(args) > 2 {
+				rate, err := sdk.NewDecFromStr(args[2])
+				if err != nil {
+					return err
+				}
+				swapFeeRate = &rate
+			}
+
+			msg := types.NewMsgCreatePool(poolCreator, uint32(poolTypeId), depositCoins, swapFeeRate)
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+// Change swap fee rate of the existing pool.
+func NewSetPoolSwapFeeRateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-swap-fee-rate [pool-id] [swap-fee-rate]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Set liquidity pool swap fee rate",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Set liquidity pool swap fee rate.
+
+Example:
+$ liquidityd tx liquidity set-swap-fee-rate 1 0.006 --from pool_governor_address
+
+This example sets swap fee rate of liquidity pool with id 1 to to 0.006.
+This action can be only taken if the pool governor address is passed as from address.
```

Comment:
Hi @barriebyron!
* **1** here is the pool id - this id uniquely determines pool which swap fee rate will be changed
* **0.006** is the new swap fee rate that will be set on the pool.

I understand that my statement is not clear enough. Maybe this would be better statement:
`This example sets a swap fee rate to 0.006 on liquidity pool 1.`

Hope that this gives more context to you to understand what I wanted to say :).